test/unittests/parser_test.cpp

```diff
@@ -119,31 +130,31 @@ TEST(parser, custom_section_nonempty)
 
 TEST(parser, functype_wrong_prefix)
 {
-    const auto section_contents = uint8_t{0x01} + from_hex("610000");
+    const auto section_contents = bytes{0x01} + from_hex("610000");
```

Comment:
Do we really need `from_hex` if we use `bytes{0x00, 0x0b}` or do we prefer to use `from_hex` everywhere instead?

Reply:
I don't mind mixing them. Hex are easier to read and work with.

If needed we can also add literals for `bytes`, e.g. `"610000"_hex`.

Reply:
Perhaps the literals would be the best for the tests?

Also thinking about introducing `make_vec` that could help in the above case.